mcs/tools/mdoc/Mono.Documentation/assembler.cs

```diff
@@ -76,22 +82,30 @@ public override void Run (IEnumerable<string> args)
 			return;
 
 		List<Provider> list = new List<Provider> ();
-		EcmaProvider ecma = null;
+		List<EcmaProvider> ecma = null;
```

Comment:
I don't understand why this is now a `List<EcmaProvider>`, particularly since only one provider will ever be added to it -- when `ecma == null`! `ecma.Add()` is only performed once; there's no need for this to be a list.